client/api_group.go

```diff
@@ -47,6 +57,13 @@ type Group interface {
 	GetObjectPolicyOfGroup(ctx context.Context, bucketName, objectName string, groupId uint64) (*permTypes.Policy, error)
 	// GetGroupPolicy get the group policy info of the user specified by principalAddr
 	GetGroupPolicy(ctx context.Context, groupName string, principalAddr string) (*permTypes.Policy, error)
+	// ListGroup get the group list by name and prefix.
```

Comment:
ListGroup get the group list by name and prefix. , return the group info which matches the pattern